media/js/firefox/hello/start.js

```diff
@@ -25,9 +25,17 @@
 
     // GA event for successful Hello conversations
     function trackGAConversationConnect() {
+        // get source from localStorage
+        var tourSource = localStorage.getItem('hello_ftu_tour_source');
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
```

Comment:
After this custom GA variable has been sent, should we reset the `localStorage` value back to `none`, else if they take the tour again directly, the source could be wrongly attributed?